cs/ccxt/base/Exchange.cs

```diff
@@ -22,21 +22,9 @@ public Exchange(object userConfig2 = null)
         var empty = new List<string>();
         transformApiNew(this.api);
 
-        this.initRestLimiter();
         this.initHttpClient();
 
-        if (this.markets != null)
-        {
-            this.setMarkets(this.markets);
-        }
         this.afterConstruct();
-
-        var isSandbox2 = this.safeBool2(this.options, "sandbox", "testnet", false);
-        var isSandbox = (isSandbox2 != null) ? (bool)isSandbox2 : false;
-        if (isSandbox)
-        {
-            this.setSandboxMode(isSandbox);
-        }
```

Comment:
`initRestRateLimiter`, `setMarkets` and `sandbox` parts are moved into `afterConstruct` (see .ts changes)